gradio/chat_interface.py

```diff
@@ -717,6 +728,13 @@ def _process_example(
                 ]
         return result
 
+    def _additional_inputs_in_examples(self):
```

Comment:
Is there a better way to detect if additional inputs are in the provided example?

Reply:
We'll need to traverse the `examples` array and check to see if the values for the additional_examples are `None`.

See https://github.com/gradio-app/gradio/blob/c45b46652d49e0a8a5d8217bbbc6ddbe4b4e91f0/gradio/helpers.py#L221 where we do something similar